src/config.rs

```diff
@@ -160,6 +160,8 @@ pub struct DataOptions {
     pub lu_res_prefix: Option<String>,
     /// The locale.xml file
     pub locale: PathBuf,
+    /// The sqlite file to serve SQL queries from
+    pub sqlite: PathBuf,
```

Comment:
Maybe add a default here so it loads without a value set?

Reply:
Right, I wasn't sure whether to. The thing with the other optional entries like `versions` is that they're truly optional/mutually exclusive, since either `res` or  `versions` is enough (as far as I understand it). However for `query`, while technically the whole thing could be done with a fancy FDB+locale vtable, this isn't implemented right now, and thus the `/query/` endpoint won't work without a SQLite set. As more frontend code is transitioned to the new API, this would mean that functionality could break if the user forgets to set the config entry, so I made it mandatory. But I can change this if you like.